src/parsing/ris_live.go

```diff
@@ -0,0 +1,46 @@
+package parsing
```

Comment:
similar to the comment on naming for the `analyzing` package, this could be `parser` or `bgpinput` or somesuch